+ source ../common/common_lib.sh
++ source /root/GitRepo/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/GitRepo/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Fri Jan  6 03:48:29 2023 - INFO  - Start environmental preparation.
+ DNF_INSTALL fcoe-utils
+ pkgs=fcoe-utils
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/GitRepo/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs fcoe-utils --node 1 --tempfile ''
+ tmpfile2=/tmp/tmps4j9g6a1
+ '[' -z '' ']'
+ tmpfile=/tmp/tmps4j9g6a1
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Fri Jan  6 03:49:30 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Fri Jan  6 03:49:31 2023 - INFO  - Start testing...
+ test_execution fcoe.service
+ service=fcoe.service
++ date '+%Y-%m-%d %T'
+ log_time='2023-01-06 03:49:31'
+ test_restart fcoe.service
+ service=fcoe.service
+ systemctl restart fcoe.service
Job for fcoe.service failed because the control process exited with error code.
See "systemctl status fcoe.service" and "journalctl -xeu fcoe.service" for details.
+ CHECK_RESULT 1 0 0 'fcoe.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='fcoe.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'fcoe.service restart failed'
+ LOG_ERROR 'fcoe.service restart failed'
+ message='fcoe.service restart failed'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'fcoe.service restart failed'
Fri Jan  6 03:49:32 2023 - ERROR - fcoe.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Fri Jan  6 03:49:33 2023 - ERROR - ../common/common_lib.sh line 34
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status fcoe.service
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'fcoe.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='fcoe.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'fcoe.service restart failed'
+ LOG_ERROR 'fcoe.service restart failed'
+ message='fcoe.service restart failed'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'fcoe.service restart failed'
Fri Jan  6 03:49:41 2023 - ERROR - fcoe.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Fri Jan  6 03:49:42 2023 - ERROR - ../common/common_lib.sh line 37
+ return 0
+ systemctl stop fcoe.service
+ CHECK_RESULT 0 0 0 'fcoe.service stop failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='fcoe.service stop failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status fcoe.service
+ grep 'Active: inactive'
+ CHECK_RESULT 1 0 0 'fcoe.service stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='fcoe.service stop failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'fcoe.service stop failed'
+ LOG_ERROR 'fcoe.service stop failed'
+ message='fcoe.service stop failed'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'fcoe.service stop failed'
Fri Jan  6 03:49:49 2023 - ERROR - fcoe.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 42'
+ message='../common/common_lib.sh line 42'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 42'
Fri Jan  6 03:49:50 2023 - ERROR - ../common/common_lib.sh line 42
+ return 0
+ systemctl start fcoe.service
Job for fcoe.service failed because the control process exited with error code.
See "systemctl status fcoe.service" and "journalctl -xeu fcoe.service" for details.
+ CHECK_RESULT 1 0 0 'fcoe.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='fcoe.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'fcoe.service start failed'
+ LOG_ERROR 'fcoe.service start failed'
+ message='fcoe.service start failed'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'fcoe.service start failed'
Fri Jan  6 03:49:52 2023 - ERROR - fcoe.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Fri Jan  6 03:49:53 2023 - ERROR - ../common/common_lib.sh line 44
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status fcoe.service
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'fcoe.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='fcoe.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'fcoe.service start failed'
+ LOG_ERROR 'fcoe.service start failed'
+ message='fcoe.service start failed'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'fcoe.service start failed'
Fri Jan  6 03:50:00 2023 - ERROR - fcoe.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Fri Jan  6 03:50:01 2023 - ERROR - ../common/common_lib.sh line 47
+ return 0
+ test_enabled fcoe.service
+ service=fcoe.service
++ systemctl is-enabled fcoe.service
+ state=disabled
+ '[' disabled == enabled ']'
+ '[' disabled == disabled ']'
++ systemctl enable fcoe.service
++ grep 'Created symlink'
++ awk '{print $3}'
+ symlink_file='/etc/systemd/system/multi-user.target.wants/fcoe.service
/etc/systemd/system/sockets.target.wants/lldpad.socket
/etc/systemd/system/sockets.target.wants/fcoemon.socket'
+ find /etc/systemd/system/multi-user.target.wants/fcoe.service /etc/systemd/system/sockets.target.wants/lldpad.socket /etc/systemd/system/sockets.target.wants/fcoemon.socket
/etc/systemd/system/multi-user.target.wants/fcoe.service
/etc/systemd/system/sockets.target.wants/lldpad.socket
/etc/systemd/system/sockets.target.wants/fcoemon.socket
+ CHECK_RESULT 0 0 0 'fcoe.service enable failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='fcoe.service enable failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl disable fcoe.service
Removed /etc/systemd/system/sockets.target.wants/lldpad.socket.
Removed /etc/systemd/system/sockets.target.wants/fcoemon.socket.
Removed /etc/systemd/system/multi-user.target.wants/fcoe.service.
+ find /etc/systemd/system/multi-user.target.wants/fcoe.service /etc/systemd/system/sockets.target.wants/lldpad.socket /etc/systemd/system/sockets.target.wants/fcoemon.socket
find: ‘/etc/systemd/system/multi-user.target.wants/fcoe.service’: No such file or directory
find: ‘/etc/systemd/system/sockets.target.wants/lldpad.socket’: No such file or directory
find: ‘/etc/systemd/system/sockets.target.wants/fcoemon.socket’: No such file or directory
+ CHECK_RESULT 1 0 1 'fcoe.service disable failed'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='fcoe.service disable failed'
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ journalctl --since '2023-01-06 03:49:31' -u fcoe.service
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
Jan 06 03:49:31 openEuler-riscv64 systemd[1]: fcoe.service: Control process exited, code=exited, status=1/FAILURE
Jan 06 03:49:31 openEuler-riscv64 systemd[1]: fcoe.service: Failed with result 'exit-code'.
Jan 06 03:49:31 openEuler-riscv64 systemd[1]: Failed to start Open-FCoE initiator daemon.
Jan 06 03:49:51 openEuler-riscv64 systemd[1]: fcoe.service: Control process exited, code=exited, status=1/FAILURE
Jan 06 03:49:51 openEuler-riscv64 systemd[1]: fcoe.service: Failed with result 'exit-code'.
Jan 06 03:49:51 openEuler-riscv64 systemd[1]: Failed to start Open-FCoE initiator daemon.
+ CHECK_RESULT 0 0 1 'There is an error message for the log of fcoe.service'
+ actual_result=0
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of fcoe.service'
+ '[' -z 0 ']'
+ '[' 1 -eq 0 ']'
+ test 0x == 0x
+ test -n 'There is an error message for the log of fcoe.service'
+ LOG_ERROR 'There is an error message for the log of fcoe.service'
+ message='There is an error message for the log of fcoe.service'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'There is an error message for the log of fcoe.service'
Fri Jan  6 03:50:16 2023 - ERROR - There is an error message for the log of fcoe.service
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 28'
+ message='../common/common_lib.sh line 28'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 28'
Fri Jan  6 03:50:17 2023 - ERROR - ../common/common_lib.sh line 28
+ return 0
+ test_reload fcoe.service
+ service=fcoe.service
+ systemctl start fcoe.service
Job for fcoe.service failed because the control process exited with error code.
See "systemctl status fcoe.service" and "journalctl -xeu fcoe.service" for details.
+ grep 'Job type reload is not applicable'
+ systemctl reload fcoe.service
Failed to reload fcoe.service: Job type reload is not applicable for unit fcoe.service.
+ CHECK_RESULT 0 0 0 'Job type reload is not applicable for unit fcoe.service'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Job type reload is not applicable for unit fcoe.service'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl status fcoe.service
+ grep 'Active: active'
+ systemctl status fcoe.service
+ grep 'inactive (dead)'
+ return 1
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Fri Jan  6 03:50:20 2023 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 6
+ test 6 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Jan  6 03:50:21 2023 - ERROR - failed to execute the case.
+ exit 6
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Fri Jan  6 03:50:22 2023 - INFO  - start environment cleanup.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmps4j9g6a1 ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmps4j9g6a1
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/GitRepo/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Fri Jan  6 03:50:48 2023 - INFO  - Finish environment cleanup!
